scripts/local_testnet/setup_time.sh

```diff
@@ -20,16 +20,21 @@ echo "slot_per_epoch=$SLOT_PER_EPOCH"
 genesis_file=$1
 
 # Update future hardforks time in the EL genesis file based on the CL genesis time
-GENESIS_TIME=$(lcli pretty-ssz --spec $SPEC_PRESET --testnet-dir $TESTNET_DIR BeaconState $TESTNET_DIR/genesis.ssz | jq | grep -Po 'genesis_time": "\K.*\d')
-echo $GENESIS_TIME
+GENESIS_TIME=$(lcli pretty-ssz --spec $SPEC_PRESET --testnet-dir $TESTNET_DIR BeaconState $TESTNET_DIR/genesis.ssz | jq | sed -n 's/.*genesis_time": "\([0-9]*\).*/\1/p')
```

Comment:
The original script outputs the `genesis_time`, so does your script - they do the same thing. Not sure what do you mean by invalid option `--P`? Does the script not work in your OS (MacOS)?